pyproject.toml

```diff
@@ -17,12 +17,12 @@ dependencies = [
     "seaborn>=0.13.2",
     "plotly>=5.23.0",
     "trimesh>=4.1.0",
-    "point-cloud-utils==0.30.4",
+    "point-cloud-utils>=0.30.0",
     "pyvista>=0.43.10,<0.44",
     "scikit_posthocs>=0.9.0",
     "kaleido>=0.2.1"
 ]
-requires-python = ">=3.11"
+requires-python = "~=3.11"
```

Comment:
"~=3.11" is equivalent to the more-readable ">=3.11,<4". I recommend the latter because people may expect `~=3.11` to mean `3.11.*`.

Reply:
based on issues I had testing in 3.12 and reading posts on its github, it seems that `point_cloud_utils` is not compatible with 3.12, at least the version that was used for the paper.

Reply:
`quilt3` seems to be missing from the dependencies list. When I tried to install separetly, got this:

```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
aiobotocore 2.14.0 requires botocore<1.35.8,>=1.35.0, but you have botocore 1.35.11 which is incompatible.
```

Running Python 3.11.9.

Reply:
Addressed in [1367bd8](https://github.com/AllenCell/EMT_data_analysis/pull/6/commits/1367bd8cb00d1236f42f7b351fe01c5f636199c2)